### DISCUSSION TITLE
0.9.2 - New Languages and Bug Fixes

### DISCUSSION THREAD

**Discussion by Timthreetwelve** [Timthreetwelve/GetMyIP#34] [Announcements]
#### Changes:
- Added: Italian language #25.
- Added: Dutch language #25.
- Added: Option to obfuscate IP address details in log.
- Added: Option to use [ipext.org](https://ipext.org) as external IP address provider.
- Fixed: Error parsing JSON. JSON was null #20.
- Fixed: Desktop shortcuts not removed after uninstalling #21.
- Updated: Inno Setup installer improvements #24, #29.
- Updated: Community Toolkit Mvvm dependency has been updated to the latest version.
- Updated: Several UI fixes and tweaks.

In addition to the application updates, [Discussions](https://github.com/Timthreetwelve/GetMyIP/discussions) have been opened. Check it out to see announcements, ask a question or share an idea.

#### Thanks:
Big thanks to everyone that contributed language translations, tracking down bugs, Inno Setup improvements, and friendly advice. Your help is invaluable. ❤️

If you wish to contribute a translation, check out the [Contribute a Translation](https://github.com/Timthreetwelve/GetMyIP/wiki/Contribute-a-Translation) topic in the Wiki. Your help would be greatly appreciated.

**Full Changelog**: https://github.com/Timthreetwelve/GetMyIP/compare/v0.9.1...v0.9.2

#### Checksums:
|File|SHA256 hash|
|----|---------------|
|Installer (exe)|`dcfa91d7046dec3b8cde90671912b5ccef52a9cd155b64a38632d374e339f861` |
|Portable (zip)|`a9b9a7a2ce857bd6805d92db106bb3a895f7c15005d1619a7265ae16030c1004` |

<hr /><em>This discussion was created from the release <a href='https://github.com/Timthreetwelve/GetMyIP/releases/tag/v0.9.2'>0.9.2 - New Languages and Bug Fixes</a>.</em>

**Comment by CMTriX** [comment]
Crossing fingers! NO JSON error anymore!

**Comment by CMTriX** [comment]
Can confirm Shortcuts are now removed correct after uninstall.

**Comment by Timthreetwelve** [comment]
Good news. Thanks!
